__tests__/index.test.js

```diff
@@ -359,4 +359,37 @@ describe('GET /api/users', () => {
         })
     })
 
+})
+
+describe('GET topic query /api/articles', () => {
+    test('takes a topic query and responds with all articles on that topic', () => {
+        return request(app)
+        .get('/api/articles')
+        .query({topic: 'cats'})
```

Comment:
We don't see this often, so long as you know what is going on here then this is really good 👍